src/App.tsx

```diff
@@ -4,15 +4,18 @@ import AboutPage from '@/components/about/About.component.tsx';
 import { LayoutComponent } from '@/components/layout/LayoutComponent.tsx';
 import NotFound from '@/components/notfound/NotFound.component.tsx';
 import ProductsPage from '@/components/product/ProductsPage.component.tsx';
+import { ProductsDataContextProvider } from '@/utils/Products';
 
 export default function App() {
     return (
-        <Routes>
-            <Route path="/" element={<LayoutComponent />}>
-                <Route index element={<AboutPage />} />
-                <Route path="products" element={<ProductsPage />} />
-                <Route path="*" element={<NotFound />} />
-            </Route>
-        </Routes>
+        <ProductsDataContextProvider>
+            <Routes>
+                <Route path="/" element={<LayoutComponent />}>
```

Comment:
Please add a link to about page to the site logo
![image](https://github.com/demasha0908/fe-react-2024/assets/98031768/f446fe2d-c838-4ca2-9a5d-ea89863e6c95)

Reply:
Fixed